<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>lib</class>
    <widget class="QWidget" name="lib">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>760</width>
                <height>470</height>
            </rect>
        </property>
        <property name="font">
            <font>
                <family>Georgia</family>
                <pointsize>14</pointsize>
            </font>
        </property>
        <property name="windowTitle">
            <string>libName</string>
        </property>
        <widget class="QListWidget" name="listWidget">
            <property name="geometry">
                <rect>
                    <x>3</x>
                    <y>3</y>
                    <width>541</width>
                    <height>61</height>
                </rect>
            </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="geometry">
                <rect>
                    <x>3</x>
                    <y>67</y>
                    <width>754</width>
                    <height>341</height>
                </rect>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
            <property name="geometry">
                <rect>
                    <x>546</x>
                    <y>2</y>
                    <width>211</width>
                    <height>63</height>
                </rect>
            </property>
            <property name="text">
                <string>Save</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
                <rect>
                    <x>2</x>
                    <y>410</y>
                    <width>755</width>
                    <height>56</height>
                </rect>
            </property>
            <property name="text">
                <string>Your templates</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
